### ISSUE TITLE
NTR: Avascular necrosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#3884]
New term request: Avascular necrosis. Definition: \
Source: \
Parent: \
Disease association:

**Comment by pnrobinson** [comment]
- [ ] Are you referring to Avascular necrosis of the capital femoral epiphysis 
https://hpo.jax.org/app/browse/term/HP:0005743

**Comment by pnrobinson** [comment]
Actually, I think "Avascular necrosis" is a more general term, and we may need to make a subhierarchy for it, see
https://en.wikipedia.org/wiki/Avascular_necrosis

**Comment by pnrobinson** [comment]
Check also how much this overlaps with the parent term Juvenile aseptic necrosis

**Comment by pnrobinson** [comment]
Actually, Aseptic necrosis already has avascular necrosis as a synonym
https://hpo.jax.org/app/browse/term/HP:0010885

**Comment by pnrobinson** [comment]
I have switched the primary label to Avascular necrosis, which is more recognizable
